tcp/server.go

```diff
@@ -49,35 +57,36 @@ func getTCProot(root string) string {
 	return root
 }
 
-//log Error if one occures
-func checkErr(err error) {
-	if err != nil {
-		qbin.Log.Errorf("TCP Server Error: %s", err.Error())
-	}
-}
-
 //handle incomming TCP Connection
 func handleClient(connTCP net.Conn, root string) {
+	qbin.Log.Debug("Handeling incomming connection...")
 	//set timeout to 5 seconds, limit requestlength to 1MB, close connection on programm close
-	connTCP.SetReadDeadline(time.Now().Add(5 * time.Second))
+	connTCP.SetReadDeadline(time.Now().Add(1500 * time.Millisecond))
```

Comment:
Please confirm: What if I send 10 bytes now, 10 bytes a second later, and 10 bytes another second later. Will all 30 bytes get read as they should?

Reply:
If not, I guess reading manually instead of using `ioutil.ReadAll()` would be better, because the deadline can be updated on every packet received.

Reply:
I need to change the reading method, because the deadline isn't reset on receive.